wp-content/themes/core/blocks/tribe/vertical-tabs/edit.js

```diff
@@ -8,12 +8,12 @@ import { Button, Flex, FlexItem, Tooltip } from '@wordpress/components';
 import { useDispatch, useSelect } from '@wordpress/data';
 import { useEffect } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
-import { Icon, link, trash } from '@wordpress/icons';
+import { Icon, trash } from '@wordpress/icons';
 
 import './editor.pcss';
 
 export default function Edit( { attributes, clientId, setAttributes } ) {
-	const blockProps = useBlockProps();
+	const blockProps = useBlockProps( { className: 'b-vertical-tabs' } );
```

Comment:
Curious why you chose to apply this change, and has it been applied to other blocks?

Reply:
WordPress has a fun habit of duplicating classes in the editor when you use server-side rendering (dynamic blocks). Doesn't seem to be happening on this block specifically, but a lot of our new blocks using this naming scheme when using dynamic blocks, so I like it better this way.